ui/src/app/workflows/components/workflows-toolbar/workflows-toolbar.tsx

```diff
@@ -0,0 +1,127 @@
+import {NotificationType} from 'argo-ui';
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+import {Workflow} from '../../../../models';
+import {AppContext, Consumer} from '../../../shared/context';
+import * as Actions from '../../../shared/workflow-actions';
+
+require('./workflows-toolbar.scss');
+
+interface WorkflowsToolbarProps {
+    selectedWorkflows: {[index: string]: Workflow};
+    loadWorkflows: () => void;
+    isDisabled: Actions.ActionDisabled;
+}
+
+interface WorkflowGroupAction {
```

Comment:
It seems like this is just extended from `WorkflowAction`. Could we just extend it? https://www.typescriptlang.org/docs/handbook/interfaces.html#extending-interfaces

Reply:
Yeah, definitely. Wasn't aware of that syntax